### ISSUE TITLE
用 CartoonGAN 及 TensorFlow 2 生成新海誠與宮崎駿動畫

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#12121]
用 CartoonGAN 及 TensorFlow 2 生成新海誠與宮崎駿動畫<br>
<br>
https://ift.tt/0mFOwyS<br>
<br>
Lee Meng<br>
<br>

<div><div><div><div>
<div>
<p>雖然我們只紀錄專屬於 CartoonGAN 的指標以及生成圖片，你完全可以運用類似的方法來監測任何想要訓練的模型。</p>
<p>最後，為了方便了解 CartoonGAN 在訓練過程的表現，我們可以事先存一組真實世界的圖片當作驗證集（Validation Set），並固定讓模型在訓練一段時間後都試著將其轉換成動漫。</p>
<p>我們可以用驗證集的轉換結果來評估模型在當下的表現。將這些圖片依時間排序後可以得到這樣的結果：</p>
</div>
</div></div></div></div><br>
<br>
<br>
<br>
<br>
<br>
via LeeMeng - All About DataScience <br>
<br>
April 21, 2025 at 01:41PM
